package.json

```diff
@@ -27,8 +27,8 @@
     "ext": "*"
   },
   "devDependencies": {
-    "@antora/cli": "^2.3.4",
-    "@antora/site-generator-default": "^2.3.4",
+    "@antora/cli": "^3.0.1",
```

Comment:
This was already done in the release branch, so does this need the latest changes merged into this `staging/TinyMCE_6.0` branch?

Reply:
yep, we have to do it in release too.

Reply:
What do you mean? As I said it's already done in the release branch, which means this is behind the release branch.